lib/details_page/widgets/shopping_cart_inkwell.dart

```diff
@@ -33,3 +48,14 @@ class ShoppingCartInkWell extends ConsumerWidget {
     );
   }
 }
+
+List<ProductViewData> _addQuantityToProduct(
+    ProductViewData product, List<ProductViewData> productsInCart) {
+  if (!productsInCart.contains(product)) {
+    product.quantity = 1;
+    productsInCart.add(product);
+  } else {
+    product.quantity++;
+  }
```

Comment:
Da forma como estava anteriormente, estava duplicando código, quando adicionava um a quantidade do produto nos dois casos
```suggestion
  if (!productsInCart.contains(product)) {
    productsInCart.add(product);
  } 
  product.quantity++;
```